src/hotspot/share/code/codeCache.cpp

```diff
@@ -1559,6 +1559,34 @@ void CodeCache::log_state(outputStream* st) {
             unallocated_capacity());
 }
 
+void CodeCache::write_perf_map(outputStream* st) {
+  MutexLocker mu(CodeCache_lock, Mutex::_no_safepoint_check_flag);
+
+  char fname[32];
+  jio_snprintf(fname, sizeof(fname), "/tmp/perf-%d.map", os::current_process_id());
```

Comment:
/tmp as the temporary directory is very unix*. 
if you don't use #ifdef LINUX, this line needs a cross-platform temp.

Reply:
I would like to take back this. I've seen that `CodeCache::write_perf_map` is wrapped by \#ifdef LINUX. 
I read the older revision of change.